README.md

```diff
@@ -1,6 +1,28 @@
 # ain-py
 
+A python version of ain-js [link to https://www.npmjs.com/package/@ainblockchain/ain-js].
```

Comment:
could you follow the md syntax?
e.g. [ain-js](ttps://www.npmjs.com/package/@ainblockchain/ain-js)

Reply:
Addressed! Thanks!